src/client/Microsoft.Identity.Client/AppConfig/ApplicationType.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Client.AppConfig
+{
+    /// <summary>
+    /// Enum to represent the type of application.
+    /// </summary>
+    internal enum ApplicationType
```

Comment:
```suggestion
    internal enum MSALApplicationType
```
We are dealing with so many apps and app types, would be good to be specific on this App Type

Reply:
`MsalClientType` ?